app/plugins/Wallet/css/wallet.css

```diff
@@ -218,9 +218,15 @@
 }
 .entry .address {
 	display: inline-block;
-	width: 80%;
+	width: calc(80% - 25px);
```

Comment:
I don't see how. The icon column is set to 25% pix and this `calc` just computes the correct width of the addresses column.